public/app/features/admin/UserOrgs.tsx

```diff
@@ -0,0 +1,264 @@
+    `,
+    label: css`
+      font-weight: 500;
+    `,
+  };
+});
+
+interface OrgRowProps extends Themeable {
+  org: UserOrg;
+  onOrgRemove: (orgId: number) => void;
+  onOrgRoleChange: (orgId: number, newRole: string) => void;
+}
+
+interface OrgRowState {
+  currentRole: string;
+  isChangingRole: boolean;
+  isRemovingFromOrg: boolean;
+}
+
+class UnThemedOrgRow extends PureComponent<OrgRowProps, OrgRowState> {
+  state = {
+    currentRole: this.props.org.role,
+    isChangingRole: false,
+    isRemovingFromOrg: false,
+  };
+
+  handleOrgRemove = () => {
+    const { org } = this.props;
+    this.props.onOrgRemove(org.orgId);
+  };
+
+  handleChangeRoleClick = () => {
+    const { org } = this.props;
+    this.setState({ isChangingRole: true, currentRole: org.role });
+  };
+
+  handleOrgRemoveClick = () => {
+    this.setState({ isRemovingFromOrg: true });
+  };
+
+  handleOrgRoleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const newRole = event.target.value;
+    this.setState({ currentRole: newRole });
+  };
+
+  handleOrgRoleSave = () => {
+    this.props.onOrgRoleChange(this.props.org.orgId, this.state.currentRole);
+  };
+
+  handleCancelClick = () => {
+    this.setState({ isChangingRole: false, isRemovingFromOrg: false });
+  };
+
+  render() {
+    const { org, theme } = this.props;
+    const { currentRole, isChangingRole, isRemovingFromOrg } = this.state;
+    const styles = getOrgRowStyles(theme);
+    const labelClass = cx('width-16', styles.label);
+
+    return (
+      <tr>
+        <td className={labelClass}>{org.name}</td>
+        {isChangingRole ? (
+          <td>
+            <div className="gf-form-select-wrapper width-8">
+              <select value={currentRole} className="gf-form-input" onChange={this.handleOrgRoleChange}>
+                {ORG_ROLES.map((option, index) => {
+                  return (
+                    <option value={option} key={`${option}-${index}`}>
+                      {option}
+                    </option>
+                  );
+                })}
+              </select>
+            </div>
+          </td>
+        ) : (
+          <td className="width-25">{org.role}</td>
+        )}
+        {!isRemovingFromOrg && (
+          <td colSpan={isChangingRole ? 2 : 1}>
+            <div className="pull-right">
+              <ConfirmButton
+                buttonText="Change role"
+                confirmText="Save"
+                onClick={this.handleChangeRoleClick}
+                onCancel={this.handleCancelClick}
+                onConfirm={this.handleOrgRoleSave}
+              />
+            </div>
+          </td>
+        )}
+        {!isChangingRole && (
+          <td colSpan={isRemovingFromOrg ? 2 : 1}>
+            <div className="pull-right">
+              <ConfirmButton
+                buttonText="Remove from organisation"
+                confirmText="Confirmn removal"
+                confirmButtonVariant="danger"
```

Comment:
Can we assume confirm button has always `danger` variant?

Reply:
No, in some cases (like Save) we need inverse button.